### PR TITLE
Add see also and content xref links

### LINKED ISSUE #24084: JS names for CSS properties
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle

### What specific section or headline is this issue about?

Notes

### What information was incorrect, unhelpful, or incomplete?

The text includes this important statement:

> CSS property values may be accessed using the getPropertyValue(propName) API or by indexing directly into the object such as obj['z-index'] or obj.zIndex.

Which likely deserves it's own page.

### What did you expect to see?

A link to a document describing https://drafts.csswg.org/cssom/#css-property-to-idl-attribute or, possibly, 

```js 
let availableProps = getComputedStyle(document.body);
let list = [];
for(cssProp in availableProps) { 
  if(!/\-/.test(cssProp) && /^[a-z]/.test(cssProp)) {
    list.push(cssProp);
  }
}
console.log(list);
``` 
as a live example in the page

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/window/getcomputedstyle`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/window/getcomputedstyle/index.md
* Last commit: https://github.com/mdn/content/commit/76264bba2d8b68b00de3eca0ad1e64c4aa77af39
* Document last modified: 2023-01-12T04:22:58.000Z

</details>

### PR DESCRIPTION
**Pull request by estelle**
fixes https://github.com/mdn/content/issues/24084

added a see also link to fix the issue, then realized there were a few other edits that could be made.

### CODE REVIEW

File: files/en-us/web/api/window/getcomputedstyle/index.md [hunk 1]
```diff
@@ -186,4 +186,5 @@ Java.
 ## See also
 
 - {{DOMxRef("window.getDefaultComputedStyle")}}
```
Comment (skyclouds2001) [inline files/en-us/web/api/window/getcomputedstyle/index.md:188]:
```suggestion
- {{DOMxRef("window.getDefaultComputedStyle()")}}
```

not relates to this, but a improvement

**Review by wbamberg — COMMENTED** [review]
I'm a bit uncertain how this is a fix for https://github.com/mdn/content/issues/24084.

I'm not that sure what the issue in https://github.com/mdn/content/issues/24084, but it seems like it's asking for some explanation of how CSS property names should be mapped to attribute names? e.g. `z-index`->`zIndex`?

But then the addition in this PR is just a link to https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/getPropertyValue, which doesn't seem to meet that at all.

Also, unrelated, I didn't understand this in the page:

> The returned [CSSStyleDeclaration](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration) object contains active values for CSS property longhand names. For example, `border-bottom-width` instead of the `border-width` and `border` [shorthand property names](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties). It is safest to query values with only longhand names like `font-size`. Shorthand names like `font` will not work with most browsers.

This works totally fine for me in Firefox and Chrome:

```js
const h1 = document.querySelector("h1"); 
const styles = window.getComputedStyle(h1);
styles["font"];
// '700 32px -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif' 
```

So what is the page saying?

**Comment by estelle** [comment]
> I'm a bit uncertain how this is a fix for #24084. I'm not that sure what the issue in #24084, but it seems like it's asking for some explanation of how CSS property names should be mapped to attribute names? e.g. `z-index`->`zIndex`?

It was my issue. I didn't want to the page to explain property name mapping. Rather, i wasnted to explain array object v. dot notation, so added that. Not saying adding mapping wouldn't help. But, that was what i was thinking when i wrote the issue.



> Also, unrelated, I didn't understand this in the page:
> 
> > The returned [CSSStyleDeclaration](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration) object contains active values for CSS property longhand names. For example, `border-bottom-width` instead of the `border-width` and `border` [shorthand property names](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties). It is safest to query values with only longhand names like `font-size`. Shorthand names like `font` will not work with most browsers.
> 
> This works totally fine for me in Firefox and Chrome:
> 
> ```js
> const h1 = document.querySelector("h1"); 
> const styles = window.getComputedStyle(h1);
> styles["font"];
> // '700 32px -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif' 
> ```
> 
> So what is the page saying?

That section has a bunch of notes related to Firefox. I was just updating that paragraph to include links to mentioned features, but I am ok with removing that. Not sure if the other FF comments should be removed, but that's a different PR scope

**Comment by estelle** [comment]
> but I am ok with removing that

I updated instead

**Comment by estelle** [comment]
@wbamberg Thank you for the review. When you have a chance, a re-review is appreciated.

**Review by wbamberg — APPROVED** [review]
👍 sorry to be slow reviewing this.

**Pull request merged**